src/converter.ts

```diff
@@ -253,6 +262,20 @@ export class Converter {
     return ret
   }
 
+  private convertFileToNotes(tpl: TemplateResult, file: File): File {
+    const ret = file.convert(this.options.output, { extension: 'txt' })
+    const comments = new Marp().render(file.buffer?.toString() || '').comments
+    if (comments.flat().length === 0) {
+      warn(`${file.relativePath()} contains no notes.`)
+      ret.buffer = Buffer.from('no notes found')
```

Comment:
Warning is great. On the other hand, I think outputting empty notes as the same format as regular notes is better. Users who wanted automated workflow by Marp CLI will become not to have to deal with invalid format to parse notes.

Reply:
addressed in b631f44a